Feature simplify configurations (#581)
RI-SE/ATOS: docs/Installation/quickstart.md

#### Prerequisites
Make sure you have installed ATOS by following the instructions in the [Installation](installation.md) section.

-------------------------

# Quick start

> This section will walk you through the steps of starting ATOS on a host machine that has never started ATOS before. 

Reviewer: ```suggestion
This section will walk you through the steps of starting the ATOS server for the first time. 
```

> * Start ATOS by running either one of the docker commands (or with "ros2 launch ..." if you have installed it locally). This guide will show ATOS running on a host computer with docker compose.

Reviewer: ```suggestion
* Start ATOS by running either one of the docker commands (or with "ros2 launch ..." if you have installed it locally). In this guide, the docker compose command is used.```

        
        /<git_repo_path>/ATOS $ docker compose up
        (or if you are not in the docker group)
        /<git_repo_path>/ATOS $ sudo -E docker compose up
       

* Make sure the folder ~/.astazero/ATOS/ was created on the host machine. Since it was created by docker you will need to change the owner to your own user. Run the following command on the host machine:

       
        sudo chown -R $USER:$USER ~/.astazero/ATOS/
        

> * Download and build the ISO_objectDemo binary from the [isoObject repo](https://github.com/RI-SE/isoObject). Follow the build instructions and execute the binary ISO_objectDemo in the build folder. (This object is not controlable but will respond to control messages from ATOS. It is used to test the connection between ATOS and the object.)

Reviewer: ```suggestion
* Download and build the client software _ISO\_objectDemo_ from the [isoObject repo](https://github.com/RI-SE/isoObject). Follow the build instructions and execute the binary ISO_objectDemo in the build folder. This client software creates a virtual test object which will respond to control messages from ATOS. This software can be used to verify the correctness of communication between server and client)
```

> * Download and build the ISO_objectDemo binary from the [isoObject repo](https://github.com/RI-SE/isoObject). Follow the build instructions and execute the binary ISO_objectDemo in the build folder. (This object is not controlable but will respond to control messages from ATOS. It is used to test the connection between ATOS and the object.)

Reviewer 2: ```suggestion
* Download and build the ISO_objectDemo binary from the [isoObject repo](https://github.com/RI-SE/isoObject). Follow the build instructions and execute the binary ISO_objectDemo in the build folder. (This object is not controllable but will respond to control messages from ATOS. It is used to test the connection between ATOS and the object.)
```

    ![Alt text](isoobject_preconnect.png)

* Open a web browser and go to [https://localhost:3443](https://localhost:3443). You should see a page warning about NET::ERR_CERT_AUTHORITY_INVALID. This is because the certificates we create are self signed. Click on "Advanced" and then "Proceed to localhost (unsafe)". Now you should see the ATOS control panel.

> 

Reviewer: ```suggestion

Repeat the above steps for the self-signed WebSocket SSL certificate [https://localhost:9090](https://localhost:9090)
```
